testcases/kernel/controllers/memcg/control/memcg_control_test.sh

```diff
@@ -116,7 +116,7 @@ result()
 cleanup()
 {
 	if [ -e $TST_PATH/mnt ]; then
-		umount $TST_PATH/mnt 2> /dev/null
+		umount -l $TST_PATH/mnt 2> /dev/null
 		rm -rf $TST_PATH/mnt
 	fi
 }
```

Comment:
Better fix would be to:

* Make sure the test process is killed
  (just kill it in unlikely case that it does not end up killed by kernel)
* Wait for the $! at the end of the testcase_1()